packages/keystatic/src/app/CollectionPage.tsx

```diff
@@ -88,6 +89,19 @@ function CollectionPageHeader(props: {
     setSearchVisible(isAboveMobile);
   }, [isAboveMobile]);
 
+  // the entries are presented in a virtualized table view, so we have to
+  // replace the default (e.g. ctrl+f) browser search behaviour
+  useEffect(() => {
+    const listener = (event: KeyboardEvent) => {
+      if (isHotkey('mod+f', event)) {
+        event.preventDefault();
+        searchRef.current?.select();
```

Comment:
Do we want to avoid prevent defaulting if the search input is already focused so people can mod+f twice if they want to open the browser search for whatever reason?

Reply:
Yeah, that sounds reasonable.

- [ ] allow browser-find when hotkey repeated whilst search focused